ARICHReco/ARICHReco.fcl

```diff
@@ -2,7 +2,24 @@ BEGIN_PROLOG
 
 standard_arichreco:
 {
-  module_type: ARICHReco
+  module_type: 	  ARICHReco
+  PD_file:        "/ARICHReco/PMTDet.root"
```

Comment:
we'll need a longer term strategy with reading in root files, so mark this up in an issue to revisit and decide how best to handle